docs/extending/effects.md

```diff
@@ -1,16 +1,18 @@
-Effects temporarily or permanently change their target such as
+Effects temporarily or permanently change attributes of their target. For example:
```

Comment:
Effects don't have to change attributes at all, they could just make the target say something weird every 3 minutes for example.